src/query_parser/v2/parser.y

```diff
@@ -1017,14 +989,12 @@ vector_attribute_list(A) ::= vector_attribute(B). {
   A.needResolve = array_append(A.needResolve, B.needResolve);
 }
 
-// Vector range queries
-expr(A) ::= modifier(B) COLON LSQB vector_range_query(C) RSQB. {
+/*** Vector range queries ***/
+expr(A) ::= modifier(B) COLON LSQB vector_range_query(C) RSQB. { // top-level parse
```

Comment:
Notice that this comment is inside the code block and therefore will be part of the `parser.c` file.